### PR TITLE
Design updates & bug fixes

### PR DESCRIPTION
**Pull request by mertkahyaoglu**

### CODE REVIEW

File: src/utils/date.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    } else if (delta < yesterday) {
+        result = "yesterday";
+    } else if (delta < week) {
+        result = Math.round(delta / day) + "d";
+    } else if (delta < month) {
+        result = Math.round(delta / week) + "w";
+    } else if (delta < year) {
+        result = Math.round(delta / month) + "mth";
+    } else {
+        result = Math.round(delta / year) + "y";
```
Comment (yannbf) [inline src/utils/date.ts:27]: Nice! Plus for not using a lib just for this haha
I am wondering if it makes the design a bit more troublesome if you were to use a more extensive form like:
`year(s) ago`, `day(s) ago` etc?

Comment (mertkahyaoglu) [reply src/utils/date.ts:27]: Yeah, that was the point. But I guess I can add `ago` at the end.

File: src/utils/date.ts [hunk 2]
```diff
@@ -12,19 +12,19 @@ function relativeDate(date: number): string {
...
+        result = Math.round(delta / week) + "w ago";
     } else if (delta < year) {
-        result = Math.round(delta / month) + "mth";
+        result = Math.round(delta / month) + "mth ago";
     } else {
-        result = Math.round(delta / year) + "y";
+        result = Math.round(delta / year) + "y ago";
     }
 
     return result;
```
Comment (yannbf) [inline src/utils/date.ts:30]:
```suggestion
    return `${result} ago`;
```

Comment (yannbf) [reply src/utils/date.ts:30]: if all possibilities append "ago", you can have that appended once in the return statement

Comment (mertkahyaoglu) [reply src/utils/date.ts:30]: Nah, it doesn't apply for `now` and `yesterday`.

**Comment by yannbf** [comment]
LGTM <3

**Pull request merged**